scripts/check_urls.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python3
+
+import os
+
+def scanDirectory(path):
+    entries = os.listdir(path)
+    for entry in entries:
+        if os.path.isfile(os.path.join(path, entry)) and ".md" in entry:
+            readFile(os.path.join(path, entry))
+        if (os.path.isdir(os.path.join(path, entry))):
+            scanDirectory(os.path.join(path, entry))
+
+import re
+import requests
+import multiprocessing as mp
+
+manager = mp.Manager()
+invalid_uris = manager.list()
+uris = []
+
+
+def readFile(path):
+    with open(path, 'r') as f:
+        for line in f.readlines():
+            result = re.search(r"\[(.*?)\]\((.*?)\)", line)
+            if(result and not "ioit://" in result.group(2)):
+                uris.append(result.group(2))
+
+def testhttpuri(uri):
+    try:
+        headers = {
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36'
+        }
+        r = requests.get(uri, headers=headers, timeout = 5)
```

Comment:
could be nice to place headers and timeout on top of file so we could configure how the scanning should be done